includes/footer.php

```diff
@@ -71,19 +67,16 @@
           </div>
 
           <div class="social-icons mb-3 d-print-none">
-            <a rel="me" href="https://nfcore.slack.com/" target="_blank" title="Slack" data-bs-toggle="tooltip" data-bs-placement="bottom">
+            <a rel="me" href="https://sangertreeoflife.slack.com/" target="_blank" title="Slack" data-bs-toggle="tooltip" data-bs-placement="bottom">
```

Comment:
ToL doesn't have a public Slack

- sangertreeoflife.slack.com is only for Sanger staff.
- darwintreeoflife.slack.com, moore-aqsym-genomics.slack.com, and genomeark.slack.com are for project members only
- assemblycuration.slack.com is open to everyone, but is mostly for assembly curation

Reply:
Should I remove the Slack?

Reply:
Yes.